luaui/Widgets/dbg_test_runner.lua

```diff
@@ -154,7 +154,7 @@ local function findTestFiles(directory, patterns, rootDirectory, result)
 		result = {}
 	end
 
-	for _, filename in ipairs(VFS.DirList(directory, "*", VFS.RAW_FIRST)) do
+	for _, filename in ipairs(VFS.DirList(directory, "*.lua", VFS.RAW_FIRST)) do
```

Comment:
is this change intended? looks out of scope.

Reply:
It's needed for the added test. Definitely possible to split it out to another PR, just awkward to enforce that it's merged first.

Reply:
No problem for me, but maybe you should comment about it in the PR description so other reviewers know it's intended. Otherwise looks like a classic debugging leftover.